lms/templates/main.html

```diff
@@ -101,6 +101,10 @@
     else:
         header_file = microsite.get_template_path('navigation.html')
 
+    if is_microsite():
+        header_extra_file_name = 'header_extra.html'
+        if microsite.get_template_path(header_extra_file_name) != header_extra_file_name:  # make sure micro site has header extra template
+            header_extra_file = microsite.get_template_path(header_extra_file_name)
```

Comment:
I wonder if it might be cleaner to just always call microsite.get_template_path('header_extra.html') without the if conditionals. Then define a blank header_extra.html file (with a python comment explaining that it is blank on purpose and that it can be overridden by microsites) in edx-platform.